Week01/Problem03/utantipongpipat_03.py

```diff
@@ -0,0 +1,11 @@
+import math
+number = 600851475143;
+prime = 2;
+biggestPrime = 2;
+#will pull out prime factor, from small one first
+while (prime < number+1 ):
+    while (number%prime == 0):
```

Comment:
Let's say the number is 27. Prime is 3. I want to say 3 is prime. Later I may check if 9 divides the number, but it won't since I use while loop. If I use 'if', the number will be 9 and 9 divide 9, and the output will be 9, which is not a prime.